stdlib/messaging/rabbitmq/src/main/java/org/ballerinalang/messaging/rabbitmq/util/ChannelUtils.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.messaging.rabbitmq.util;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.messaging.rabbitmq.RabbitMQConstants;
+import org.ballerinalang.messaging.rabbitmq.RabbitMQUtils;
+import org.ballerinalang.model.values.BInteger;
+import org.ballerinalang.model.values.BMap;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.util.exceptions.BallerinaException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Util class for RabbitMQ Channel handling.
+ */
+public class ChannelUtils {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ChannelUtils.class);
+
+    /**
+     * Creates a RabbitMQ AMQ Channel.
+     *
+     * @param connection RabbitMQ Connection object.
+     * @return RabbitMQ Channel object.
+     */
+    public static Channel createChannel(Connection connection) {
+        try {
+            return connection.createChannel();
+        } catch (IOException exception) {
+            String errorMessage = "An error occurred while creating the channel";
+            LOGGER.error(errorMessage, exception);
+            throw new BallerinaException(errorMessage + " " + exception.getMessage(), exception);
```

Comment:
Instead of appending the space shall we append space to the end of `errorMessage`? Check all the places